LICENSE.txt

```diff
@@ -2,11 +2,6 @@ Copyright (c) 2019- Hercules Labs OÜ
 
 All rights reserved.
 
-Copyright (c) 2014-2018 John Otander for pixyll theme
-Copyright (c) 2014 Daniel Eden for animate.css
-Copyright (c) 2014 Brent Jackson for Basscss
-Copyright (c) 2013 Twitter, Inc for CSS copied from Bootstrap
```

Comment:
It seems like we must restore this. We don't own the all the rights.
Also we should clarify that imported content (other repos) and the docs directory are not included in this license.

Reply:
There's no code (except for ./build), so I'm not sure if this license even applies - we should license the content.

Reply:
The MIT license explicitly mentions documentation, so that's tricky.
ASF recommends to use Apache2 for docs too, so for our open source tools that seems appropriate. For our other documentation, I don't think we need to give away any permissions. For example, GitHub explicitly 'reserves all its rights' in its API docs. Typeclasses.com only states that the copyright is theirs, which a less paranoid but equally redundant way of saying the same thing. [All_rights_reserved#Obsolescence](https://en.wikipedia.org/wiki/All_rights_reserved#Obsolescence).

So I agree that this license doesn't apply, but I don't think we need or should grant any extra rights to our readers.
Let's delete this file.